vendor/src/github.com/microsoft/hcsshim/layerutils.go

```diff
@@ -84,9 +85,14 @@ func layerPathsToDescriptors(parentLayerPaths []string) ([]WC_LAYER_DESCRIPTOR,
 	var layers []WC_LAYER_DESCRIPTOR
 
 	for i := 0; i < len(parentLayerPaths); i++ {
-		// Create a layer descriptor, using the folder path
+		// Create a layer descriptor, using the folder name
```

Comment:
I think you should ignore this for now and wait for the graph driver refactor PR that is blocked behind this PR. Stefan has a change that unifies more of the Windows and Unix graph package but relies on the revendoring of hcsshim, so he can't submit the PR until this is merged.